src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_StringConcat.cs

```diff
@@ -184,9 +196,7 @@ private bool TryExtractStringConcatArgs(BoundExpression lowered, out ImmutableAr
                     }
                     break;
 
-                case BoundKind.NullCoalescingOperator:
-                    var boundCoalesce = (BoundNullCoalescingOperator)lowered;
-
+                case BoundNullCoalescingOperator boundCoalesce:
```

Comment:
>case BoundNullCoalescingOperator boundCoalesce: [](http://example.com/codeflow?start=16&length=47)

Please place this case back in its original place in order to reduce unnecessary diffs

Reply:
> Please place this case back in its original place in order to reduce unnecessary diffs 

Scratch that. I was looking at the diff between commits 24 and 32, and the case was moved there. However, looking at the diff against the base, it looks like the case was actually placed back into its original spot.

Reply:
Yeah, diff got a bit messy because I moved a bunch of code, so it seemed like cases were reordered